### DISCUSSION TITLE
Deepstream docker is not working properly

### DISCUSSION THREAD

**Discussion by aozdemir** [OE4T/tegra-demo-distro#173] [Q&A] [answered]
I build demo image on hardknott branch because my app working with L4T R32.5.1. i pull nvidia docker sample image and test with deepstream-app but I am getting :

```
root@081ca480e0a5:/opt/nvidia/deepstream/deepstream-5.1# /opt/nvidia/deepstream/deepstream-5.1/bin/deepstream-app -c samples/configs/deepstream-app/source30_1080p_dec_infer-resnet_tiled_display_int8.txt
/opt/nvidia/deepstream/deepstream-5.1/bin/deepstream-app: error while loading shared libraries: /usr/lib/aarch64-linux-gnu/libnvinfer.so.7: file too short
```

I used full demo image but should I install something extra? I saw some issues about deepstream but I assume these are related L4T R32.56.0

**Comment by madisongh** [comment, marked as answer]
Searching around for "docker" and "file too short" errors, it sounds like maybe you're running out of space? I also got a hit on an NVIDIA dev forum post where someone got that error because the mistakenly loaded the x86-64 container instead of the Jetson one.

**Reply by aozdemir** [reply]
i solved the problem is it because i am not running docker as nvidia container. Thank you for answers.

**Comment by dwalkes** [comment]
I think this can happen when there's a problem with the .csv mapping into the container, for instance the source file in the rootfs doesn't exist at the expected path.  Inside the container this shows up like a 0 sized file as I recall
